testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/suite/suites/SuiteDeltaLakeDatabricks104.java

```diff
@@ -31,7 +31,7 @@ public List<SuiteTestRun> getTestRuns(EnvironmentConfig config)
     {
         return ImmutableList.of(
                 testOnEnvironment(EnvSinglenodeDeltaLakeDatabricks104.class)
-                        .withGroups("configured_features", "delta-lake-databricks")
+                        .withGroups("configured_features", "delta-lake-databricks-104")
                         .withExcludedGroups("delta-lake-exclude-104")
```

Comment:
Do we need to have exclude if we need to explicitly add test to `delta-lake-databricks-104` group?